documentation/docs/connectors/bamboo-relay.md

```diff
@@ -16,12 +16,22 @@ Because Bamboo Relay is a decentralized exchange, you will need an independent c
 
 The Bamboo Relay connector supports two modes of operation, [open order book](https://0x.org/wiki#Open-Orderbook) and [coordinated order book](https://github.com/0xProject/0x-protocol-specification/blob/master/v2/coordinator-specification.md).
 
+For maximum order visibility it is advised to use the open order book mode at this time.
```

Comment:
Considering mentioning that mode is on by default, and if users want to change to coordinated mode, they can adjust the `bamboo_relay_use_coordinator` parameter in `conf_global.yml` in the `/conf` directory.